ui/raidboss/data/07-dt/dungeon/mistwake.ts

```diff
@@ -0,0 +1,129 @@
+        },
+      },
+    },
+    {
+      id: 'Mistwake Treno Catoblepas Thunder III',
+      type: 'StartsUsing',
+      netRegex: { source: 'Treno Catoblepas', id: 'A941' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Mistwake Treno Catoblepas Ray of Lightning',
+      type: 'StartsUsing',
+      // TODO: should this be custom to indicate avoiding the rocks?
+      netRegex: { source: 'Treno Catoblepas', id: 'AF19' },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Mistwake Treno Catoblepas Petribreath',
+      type: 'StartsUsing',
+      netRegex: { source: 'Treno Catoblepas', id: 'A947', capture: false },
+      response: Responses.awayFromFront(),
+    },
+    // -------------------------- Amdusias --------------------------
+    {
+      id: 'Mistwake Treno Amdusias Thunderclap Concerto Behind',
+      type: 'StartsUsing',
+      netRegex: { source: 'Amdusias', id: 'B118', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Mistwake Treno Amdusias Thunderclap Concerto Front',
+      type: 'StartsUsing',
+      netRegex: { source: 'Amdusias', id: 'B11D', capture: false },
+      response: Responses.goFront(),
+    },
+    {
+      id: 'Mistwake Treno Amdusias Thunder IV',
+      type: 'StartsUsing',
+      netRegex: { source: 'Amdusias', id: 'B126', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Mistwake Treno Amdusias Shockbolt',
+      type: 'StartsUsing',
+      // Paired with B12B, which happens first, but does not target
+      netRegex: { source: 'Amdusias', id: 'B12C' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Mistwake Treno Amdusias Thunder III',
+      type: 'StartsUsing',
+      // Paired with B128, which happens first, but does not target
+      netRegex: { source: 'Amdusias', id: 'B129' },
+      response: Responses.stackMarkerOn(),
+    },
+    // ---------------------- Thundergust Griffin ----------------------
+    {
+      id: 'Mistwake Thundergust Griffin Thunderspark',
+      type: 'StartsUsing',
+      netRegex: { source: 'Thundergust Griffin', id: 'B0EB', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Mistwake Thundergust Griffin High Volts',
+      type: 'StartsUsing',
+      netRegex: { source: 'Thundergust Griffin', id: 'B0EC', capture: false },
+      response: Responses.spread(),
+    },
+    {
+      id: 'Mistwake Thundergust Griffin Golden Talons',
+      type: 'StartsUsing',
+      netRegex: { source: 'Thundergust Griffin', id: 'B0F9' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Mistwake Thundergust Griffin Fulgurous Fall',
+      type: 'StartsUsing',
+      // TODO: should we use a custom callout to inform about upcoming
+      // electrogenetic force line through center?
+      netRegex: { source: 'Thundergust Griffin', id: 'B0F5', capture: false },
+      response: Responses.knockback(),
```

Comment:
Typical practice has been that we don't call clearly visible AoE indicators on the ground if there's plenty of time to resolve them. There have been exceptions here and there across the project's lifetime, but so far in 6-7 runs of this dungeon I haven't seen anyone have an issue with the follow-up, so I think this is fine as it is.

Reply:
Fair. I can remove this one.